### PR TITLE
Blockhash 0.1 (new formula)

### PR DESCRIPTION
**Pull request by bobobo1618**
Created new formula for the C implementation of http://blockhash.io, a
perceptual image hash library and command line tool.

### CODE REVIEW

**Comment by bobobo1618** [comment]
Sorry if the test isn't enough. I'm not sure where to get a decent image to hash and if the time it takes to hash it would be considered too long for a test.

File: Library/Formula/blockhash.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Blockhash < Formula
+  homepage "http://blockhash.io/"
+  head "https://github.com/commonsmachinery/blockhash.git"
+  url "https://github.com/commonsmachinery/blockhash/archive/0.1.tar.gz"
+  sha256 "aef300f39be2cbc1b508af15d7ddb5b851b671b27680d8b7ab1d043cc0369893"
+
+  depends_on 'pkg-config' => :build
```
Comment (tdsmith) [inline Library/Formula/blockhash.rb]: nit: double quotes, please

**Comment by tdsmith** [comment]
You could download something from https://github.com/commonsmachinery/blockhash/tree/master/testdata as a `resource` in the test block.

**Comment by tdsmith** [comment]
`unarj.rb` has an example of the resource test pattern I mentioned above ^

**Comment by bobobo1618** [comment]
Added a new test conforming roughly to that pattern (I'm using Popen and comparing the output since Blockhash doesn't return a meaningful response code) that seems to work.

**Comment by tdsmith** [comment]
Pulled, with a couple tweaks to the test. Thanks for the pull request!

**Pull request closed without merging**